pkg/kube/client.go

```diff
@@ -26,7 +26,7 @@ import (
 	"strings"
 	"time"
 
-	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/evanphx/json-patch"
```

Comment:
i has restore import, and alse build sucess. squash commits after review fainsh. thanks